src/rules/fenced-code-meta.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @fileoverview Rule to require or disallow metadata for fenced code blocks.
+ * @author TKDev7
+ */
+
+//-----------------------------------------------------------------------------
+// Type Definitions
+//-----------------------------------------------------------------------------
+
+/**
+ * @import { MarkdownRuleDefinition } from "../types.js";
+ * @typedef {"missingMetadata" | "disallowedMetadata"} FencedCodeMetaMessageIds
+ * @typedef {["always" | "never"]} FencedCodeMetaOptions
+ * @typedef {MarkdownRuleDefinition<{ RuleOptions: FencedCodeMetaOptions, MessageIds: FencedCodeMetaMessageIds }>} FencedCodeMetaRuleDefinition
+ */
+
+//-----------------------------------------------------------------------------
+// Rule Definition
+//-----------------------------------------------------------------------------
+
+/** @type {FencedCodeMetaRuleDefinition} */
+export default {
+	meta: {
+		type: "problem",
+
+		docs: {
+			recommended: false,
+			description: "Require or disallow metadata for fenced code blocks",
+			url: "https://github.com/eslint/markdown/blob/main/docs/rules/fenced-code-meta.md",
+		},
+
+		messages: {
+			missingMetadata: "Missing code block metadata.",
+			disallowedMetadata: "Code block metadata is not allowed.",
+		},
+
+		schema: [
+			{
+				enum: ["always", "never"],
+			},
+		],
```

Comment:
I suggest we define a schema option like `{ required: "always" | "never" }` or `{ required: boolean }`. This approach would make it easier to extend the options in the future without introducing breaking changes. I’d like to hear the team’s thoughts on this.

Reply:
I think using `"always"` and `"never"` makes sense. There are plenty of rules that follow this pattern.